asr-python-advanced-finetune-am-conformer-ctc-tao-finetuning.ipynb

```diff
@@ -0,0 +1,739 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src=\"http://developer.download.nvidia.com/notebooks/dlsw-notebooks/riva_asr_asr-python-advanced-finetune-am-conformer-ctc-tao-finetuning/nvidia_logo.png\" style=\"width: 90px; float: right;\">\n",
+    "\n",
+    "# How to fine-tune a Riva ASR Acoustic Model (Conformer-CTC) with TAO Toolkit\n",
+    "This tutorial walks you through how to fine-tune a Riva ASR acoustic model (Conformer-CTC) with TAO Toolkit."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## NVIDIA Riva Overview\n",
+    "\n",
+    "NVIDIA Riva is a GPU-accelerated SDK for building speech AI applications that are customized for your use case and deliver real-time performance. <br/>\n",
+    "Riva offers a rich set of speech and natural language understanding services such as:\n",
+    "\n",
+    "- Automated speech recognition (ASR). \n",
+    "- Text-to-Speech synthesis (TTS). \n",
+    "- A collection of natural language processing (NLP) services, such as named entity recognition (NER), punctuation, and intent classification.\n",
+    "\n",
+    "In this tutorial, we will fine-tune a Riva ASR acoustic model (Conformer-CTC) with TAO Toolkit. <br> \n",
+    "To understand the basics of Riva ASR APIs, refer to [Getting started with Riva ASR in Python](https://github.com/nvidia-riva/tutorials/blob/dev/22.04/asr-python-basics.ipynb). <br>\n",
```

Comment:
The link (https://github.com/nvidia-riva/tutorials/blob/dev/22.04/asr-python-basics.ipynb) points to the 22.04 release. Does this need to be updated to the current 22.06 release?

Reply:
It needs to be updated to the stable branch. I'll mark this conversation (and all related ones) as resolved once I've updated all of the links appropriately.

Reply:
Fixed.